some.json

```diff
@@ -0,0 +1,13 @@
+{
+  "email": "",
+  "prBoard": "",
+  "comment": {
+    "title": "",
+    "description": "",
```

Comment:
/ticket this should be fixed

Reply:
Here's your ticket: [this should be fixed](https://ticketgen19.atlassian.net/browse/IKKI-35)

Reply:
Here's your ticket: [this should be fixed](https://ticketgen19.atlassian.net/browse/IKKI-36)